src/libsyntax/feature_gate.rs

```diff
@@ -237,7 +237,7 @@ const KNOWN_FEATURES: &'static [(&'static str, &'static str, Option<u32>, Status
     ("stmt_expr_attributes", "1.6.0", Some(15701), Active),
 
     // Allows `#[deprecated]` attribute
-    ("deprecated", "1.6.0", Some(29935), Active),
+    ("deprecated", "1.6.0", Some(29935), Accepted),
```

Comment:
This should be bumped to 1.9.0. "The version numbers here correspond to the version in which the current status
// was set".